nikola/plugins/task/pages.py

```diff
@@ -55,6 +65,12 @@ def gen_tasks(self):
                 else:
                     context = {'pagekind': ['story_page', 'page_page']}
                 for task in self.site.generic_page_renderer(lang, post, kw["filters"], context):
+                    if task['name'] == index_paths[lang]:
+                        # Issue 3022
+                        LOGGER.error(
+                            "Post {0!r}: output path ({1}) conflicts with the blog index ({2}). "
+                            "Please change INDEX_PATH or disable index generation.".format(
+                                post.source_path, task['name'], index_paths[lang]))
```

Comment:
Maybe mention `DISABLE_INDEXES_PLUGIN_INDEX_AND_ATOM_FEED` as a way to disable index generation. That should make it easier for users to react on this message. (And they also see that this affects Atom feed generation.)

Reply:
Speaking of which, this config setting has a terrible name which should be changed in v8.